app/src/main/java/org/oppia/android/app/topic/conceptcard/ConceptCardFragmentPresenter.kt

```diff
@@ -63,11 +63,9 @@ class ConceptCardFragmentPresenter @Inject constructor(
     viewModel.conceptCardLiveData.observe(
       fragment,
       Observer {
-        val explanation = htmlParserFactory
+        view.text = htmlParserFactory
           .create(resourceBucketName, entityType, skillId, /* imageCenterAlign= */true)
```

Comment:
```suggestion
          .create(resourceBucketName, entityType, skillId, imageCenterAlign = true)
```

Since this is a Kotlin function, prefer using named arguments.

Reply:
DONE